dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -43,7 +43,8 @@ export const LiveBlock = ({
 	return (
 		<LiveBlockContainer
 			id={block.id}
-			borderColour={palette.border.liveBlock}
+			format={format}
+			supportsDarkMode={true}
```

Comment:
I suspect you want this to be `false`? Dotcom doesn't support dark mode yet.

Reply:
I'm going to add a `supportsDarkMode` prop to this component, default it to false,  and pass it down from AR